catalog/core/catalog-core-validationfilterplugin/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+
+ -->
+<metatype:MetaData xmlns:metatype="http://www.osgi.org/xmlns/metatype/v1.0.0">
+
+
+    <OCD name="Validation Filter Plugin"
+         id="org.codice.ddf.catalog.plugin.validationfilter.ValidationFilterPlugin">
+        <AD
+                description="Mapping of SECURITY attribute to user attribute."
+                name="Attribute map" id="attributeMap" required="false" type="String"
+                default="invalid-state=system-admin" cardinality="100"/>
+        <AD
+                description="Sets whether metacards with validation errors are filtered."
+                name="Show errors" id="showErrors" type="Boolean"
+                default="false"/>
+        <AD
+                description="Sets whether metacards with validation warnings are filtered."
+                name="Show warnings" id="showWarnings" type="Boolean"
```

Comment:
@harrison-tarr mentioned in the Codice ticket, this is a good opportunity to rename these flags to more friendly names like "Show cards with validation warnings" if possible